### ISSUE TITLE
Inconsistency between the <sourceDesc> definiction and the example in version 0.9.3

### ISSUE THREAD

**Issue by daliboris** [DARIAH-ERIC/lexicalresources#217]
By definition, a `<sourceDesc>` element can only contain `<listBibl>` elements as children, but the example in the [definition](https://dariah-eric.github.io/lexicalresources/pages/TEILex0/spec.html#TEI.sourceDesc) contains a `<biblStruct>` element as a child:

```xml
<sourceDesc>
  <biblStruct>
     <listBibl type="dictionaries">
        <biblStruct>
           ...
        </biblStruct>
     </listBibl>
  </biblStruct>
</sourceDesc>
```

**Comment by laurentromary** [comment]
Thanks for pointing to this. It looks like a clear typo to me. The outer <biblStruct> should disappear.

**Comment by ttasovac** [comment]
Indeed. Boris, I'd like to suggest that you submit this change along with #214 so that you can refresh your memory. I will help you with any questions, but you should give it a try.

**Comment by ttasovac** [comment]
@daliboris will you be able to take care of this? I will definitely help you, but I'd very much like you to refresh your memory about how to deal with the schema generation etc...
